Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -720,6 +804,53 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
 
     #endregion
 
+    #region Runtime Gizmo Utils
+
+    /// <summary>
+    /// Draws a simple XYZ-cross position gizmo at the target position, whose size is
+    /// scaled relative to the main camera's distance to the target position (for reliable
+    /// visibility).
+    /// 
+    /// You can also provide a color argument and lerp coefficient towards that color from
+    /// the axes' default colors (red, green, blue). Colors are lerped in HSV space.
+    /// </summary>
+    public static void DrawPosition(this RuntimeGizmoDrawer drawer, Vector3 pos,
+                                    Color lerpColor, float lerpCoeff) {
+      float targetScale = 0.06f; // 6 cm at 1m away.
+
+      var mainCam = Camera.main;
+      if (mainCam != null) {
+        float camDistance = Vector3.Distance(pos, mainCam.transform.position);
```

Comment:
Probably want to include the current gizmo matrix into this equation here.  If the user has changed the matrix the position might not represent the actual world position.

Reply:
Camera distance now calculated as `drawer.matrix * pos` to get it into world space